app/src/main/AndroidManifest.xml

```diff
@@ -60,7 +60,7 @@
       android:theme="@style/Theme.AndroidIDE"/>
     <activity
       android:name=".activities.editor.EditorActivityKt"
-      android:configChanges="orientation|screenSize"
+      android:configChanges="uiMode|orientation|screenSize"

```

Comment:
I just checked here, I think that applications that use the view system should actually interfere, I have only tested this feature in apps with Jetpack compose where it does not affect it. 
So I think activating this only in the terminal shouldn't be a problem, since the theme there is always the same